<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewWindow</class>
 <widget class="QDialog" name="NewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <widget class="QWidget" name="back1" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>571</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgb(55, 55, 55);
	border-radius:20px;
}</string>
   </property>
   <widget class="QWidget" name="back_login2" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>431</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgb(65, 65, 65);
	border-radius:15px;
}</string>
    </property>
    <widget class="QWidget" name="backUserName" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>351</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	
	background-color:rgb(88, 88, 88);
	border-radius:10px;
}</string>
     </property>
     <widget class="QLabel" name="txtUserName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	background-color: rgb(88, 88, 88);
	color:rgb(255, 213, 215);
}</string>
      </property>
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="usernameEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>97</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color:rgb(255, 213, 215);
	border-radius:10px;
	border-color: rgb(53, 69, 93);
	padding-left:10px;
	padding-right:10px;
	color:black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>25</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>217</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color:rgb(255, 213, 215);
	border-radius:10px;
	border-color: rgb(53, 69, 93);
	padding-left:10px;
	padding-right:10px;
	color:black;
}</string>
     </property>
     <property name="maxLength">
      <number>25</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="backPassword" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>351</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	
	background-color:rgb(88, 88, 88);
	border-radius:10px;
}</string>
     </property>
     <widget class="QLabel" name="txtPassword">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	background-color: rgb(88, 88, 88);
	color:rgb(255, 213, 215);
}</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="confirmPasswordEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>337</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color:rgb(255, 213, 215);
	border-radius:10px;
	border-color: rgb(53, 69, 93);
	padding-left:10px;
	padding-right:10px;
	color:black;
}</string>
     </property>
     <property name="maxLength">
      <number>25</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="backConfirm" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>351</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	
	background-color:rgb(88, 88, 88);
	border-radius:10px;
}</string>
     </property>
     <widget class="QLabel" name="txtConfirm">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	background-color: rgb(88, 88, 88);
	color:rgb(255, 213, 215);
}</string>
      </property>
      <property name="text">
       <string>Confirm Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>410</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:rgb(255, 131, 133);
	border-radius:25px;
	
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBack">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>410</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:rgb(152, 96, 100);
	border-radius:25px;
	border-color: rgb(53, 69, 93);
	
	color:gold;
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <zorder>backConfirm</zorder>
    <zorder>backPassword</zorder>
    <zorder>backUserName</zorder>
    <zorder>usernameEdit</zorder>
    <zorder>passwordEdit</zorder>
    <zorder>confirmPasswordEdit</zorder>
    <zorder>pushButton</zorder>
    <zorder>btnBack</zorder>
   </widget>
   <widget class="QLabel" name="txtMain">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poor Richard</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="backMain" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>671</width>
     <height>781</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ROG Fonts</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgb(36, 36, 36);
}</string>
   </property>
  </widget>
  <zorder>backMain</zorder>
  <zorder>back1</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
